treeherder/etl/mixins.py

```diff
@@ -121,50 +119,7 @@ def load(self, result_sets, project):
                 logger.error("ResultSet loading failed: {0}".format(message['message']))
 
 
-@python_2_unicode_compatible
```

Comment:
refactored these out to make them reusable, since this is really nice handling of the errors.